rastervision_pytorch_backend/rastervision/pytorch_backend/examples/chip_classification/spacenet_rio.py

```diff
@@ -174,7 +174,11 @@ def make_scene(scene_info) -> SceneConfig:
         external_loss_def=external_loss_def)
 
     backend = PyTorchChipClassificationConfig(
-        data=data, model=model, solver=solver, test_mode=test, log_tensorboard=True,
+        data=data,
+        model=model,
+        solver=solver,
+        test_mode=test,
+        log_tensorboard=True,
```

Comment:
Formatting fix to satisfy the format checker.

Reply:
Fixed in #1245.

Reply:
Thanks @AdeelH! I'll drop this commit.